agent/security_helper.py

```diff
@@ -61,7 +61,7 @@ def netstat_scan():
         'remote_address': c.raddr,
         'status': c.status if c.type == socket.SOCK_STREAM else None,
         'pid': c.pid
-    } for c in connections if c.raddr and c.laddr]
+    } for c in connections]
```

Comment:
you've stripped out too much - the check for `c.raddr` not empty is still required

Reply:
@rptrchv But API server now handles this case, doesn't it?
In the future we won't use nmap and instead we'll get open ports from netstat report. This is where raddr will be empty.

Reply:
> But API server now handles this case, doesn't it?
yes, but there's no need to transmit useless data

Reply:
@rptrchv it's not useless, as I've said above.